packages/twenty-server/src/engine/workspace-manager/workspace-manager.service.ts

```diff
@@ -226,4 +187,35 @@ export class WorkspaceManagerService {
       defaultAgentId: agent.id,
     });
   }
+
+  private async setupDefaultRoles(
+    workspaceId: string,
+    userId: string,
+  ): Promise<void> {
+    const memberRole = await this.roleRepository.findOne({
+      where: { label: MEMBER_ROLE_LABEL, workspaceId },
```

Comment:
Are we adding standardId column to objects @prastoin? When we do we should also migrate this as filtering by label is unreliable. Or if you have instructions on how to do this today already, let us know